who_l3_smart_tools/utils/counter.py

```diff
@@ -1,12 +1,36 @@
 class Counter:
+    """
+    A simple counter class that keeps track of a value and provides methods to access
+    the current value and get the next value.
+
+    Attributes:
+        __value (int): The current value of the counter.
```

Comment:
This should not be documented, since the `__` means this attribute winds up mangled and is not intended to be manipulated outside of the class itself.